app/2.0/docs/tools/deploy.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Deploy your app
```

Comment:
For example, if we were to retitle this PRPL pattern, we could remove the first L2 head ("PRPL pattern") and remove the intro paragraphs leading up to it that talk about serving.

Then the "Serve your app" page could slim down to just the first few paragraphs here, along with some pointers to different pages in /tools/ and maybe a few notes on differential serving.